test/Test/Main.js

```diff
@@ -45,3 +45,8 @@ exports.mainImpl = function(showNumber) {
   };
 };
 
+exports.throwErr = function(msg){
```

Comment:
Sorry to be a pain, but can we keep the style consistent with the rest of the file here please? I'm referring to having a space between `function(arg)` and the opening curly brace, and using semicolons at the end of lines (even if they're not strictly necessary).